File: linear_search.py
array_to_search_through = []
for number in range(1, 1001):
    array_to_search_through.append(number)

def linear_search(value_to_find, array_to_search_through):
     for i in range(len(array_to_search_through)):
         if value_to_find ==array_to_search_through :
           print(i)
           return i


    

def linear_search_global(value_to_find, array_to_search_through):
    index = []
    for i in range(len(array_to_search_through)):
        if value_to_find ==array_to_search_through :
            index.append(i)
        return index

File: js/linearSearch.js
const arrayToSearchThrough = [];
for (let i = 1; i <= 1000; i++) {
    arrayToSearchThrough.push(i);
}

exports.linearSearch = function(valueToFind, arrayToSearchThrough) {
 for(let i = 0; i < arrayToSearchThrough.length; i++){
    if(valueToFind === arrayToSearchThrough[i]){
        return i
    }else{
        return undefind
    }
 }
};

exports.linearSearchGlobal = function(valueToFind, arrayToSearchThrough) {
    let index = []
    for(let i = 0; i < arrayToSearchThrough.length; i++){
        if(valueToFind === arrayToSearchThrough[i]){
            index.push(i)
        }else{
            return undefind
        }
     }
};